include/common_structs.h

```diff
@@ -65,28 +65,32 @@ typedef enum {
     /* 0x10 */ NUM_TIME_TRIAL_DATA
 } TIME_TRIAL_DATA_INDEX;
 
+/**
+ * @brief The different types of surface in the game.
+*/
 enum SURFACE_TYPE {
+    /* -0x1 */ SURFACE_DEFAULT = -1,
     /* 0x00 */ AIRBORNE,
-    /* 0x01 */ PAVEMENT, // Luigi's Raceway, Toad's Turnpike, Koopa Troop beach shortcut tunnel, Mario Raceway, Royal Raceway, Rainbow Road, Block Fort, Double Deck, Skyscraper
-    /* 0x02 */ SAND_ONE, // Luigi's Raceway, Moo Moo Farm, Kalimiari Desert on course, Choco Mountain, Wario Stadium, DK Jungle on course, Yoshi Valley
-    /* 0x03 */ SAND_TWO, // Koopa Troopa Beach light color, Royal Raceway
-    /* 0x04 */ STONE_TWO, // Royal Raceway castle entrance, Bowser's Castle
-    /* 0x05 */ SNOW_ONE, // Frappe Snowland on course, Sherber Land tunnel
-    /* 0x06 */ STONE_ONE, // Royal Raceway castle bridges (even the wooden one), Banshee's Boardwalk, Big Donut
-    /* 0x07 */ SAND_FIVE, // Mario Raceway
+    /* 0x01 */ BITUMEN, // Luigi's Raceway, Toad's Turnpike, Koopa Troop beach shortcut tunnel, Mario Raceway, Royal Raceway, Rainbow Road, Block Fort, Double Deck, Skyscraper
+    /* 0x02 */ DIRT, // Luigi's Raceway, Moo Moo Farm, Kalimiari Desert on course, Choco Mountain, Wario Stadium, DK Jungle on course, Yoshi Valley
+    /* 0x03 */ SAND, // Koopa Troopa Beach light color, Royal Raceway
+    /* 0x04 */ STONE, // Royal Raceway castle entrance, Bowser's Castle
+    /* 0x05 */ SNOW, // Frappe Snowland on course, Sherber Land tunnel
+    /* 0x06 */ BRIDGE, // Royal Raceway castle bridges (even the wooden one), Banshee's Boardwalk, Big Donut
+    /* 0x07 */ SAND_OFF_ROAD, // Mario Raceway
     /* 0x08 */ GRASS, // Luigi's Raceway, Mario Raceway, Royal Raceway, Bowser's Castle, DK Jungle, Yoshi Valley
     /* 0x09 */ ICE, // Sherbert Land
-    /* 0x0A */ SAND_THREE, // Koop Troopa Beach dark color
-    /* 0x0B */ SNOW_TWO, // Frappe Snowland off course
+    /* 0x0A */ WET_SAND, // Koop Troopa Beach dark color
+    /* 0x0B */ SNOW_OFF_ROAD, // Frappe Snowland off course
     /* 0x0C */ CLIFF, // Koopa Troopa Beach, Choco Mountain
-    /* 0x0D */ SAND_FOUR, // Kalimari Desert off course
+    /* 0x0D */ DIRT_OFF_ROAD, // Kalimari Desert off course
     /* 0x0E */ TRAIN_TRACK, // Kalimari Desert
-    /* 0x0F */ DIRT, // DK Jungle cave
-    /* 0x10 */ WOOD_TWO, // Bowser's Castle bridge 2, DK Jungle bridge
-    /* 0x11 */ WOOD_ONE, // Frappe Snowland bridge, Bowser's Castle bridge 1,3, Yoshi Valley bridge 2
-    /* 0xFC */ BOOST_PAD_TWO = 0xFC, // DK Jungle
+    /* 0x0F */ CAVE, // DK Jungle cave
+    /* 0x10 */ ROPE_BRIDGE, // Bowser's Castle bridge 2, DK Jungle bridge
+    /* 0x11 */ WOOD_BRIDGE, // Frappe Snowland bridge, Bowser's Castle bridge 1,3, Yoshi Valley bridge 2
+    /* 0xFC */ BOOST_RAMP = 0xFC, // DK Jungle
     /* 0xFD */ OUT_OF_BOUNDS, // DK Jungle river island
-    /* 0xFE */ BOOST_PAD_ONE, // Royal Raceway
+    /* 0xFE */ BOOST_PAD, // Royal Raceway
```

Comment:
How do you feel about `GRAVITY_RAMP`? Cause it gives you slightly less gravity

Reply:
I think some ambiguous can be made with gravity where in game we don't have 0-gravity or not so much related to gravity it's a very small details and people we see it the first time can understand something totally different

Reply:
Hmm... Not quite a fan of having two differing terms that mean the same thing: boost_ramp and boost_pad.
How about boost_ramp2 or boost_ramp_fly or _float or something?